src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.CreateToken.cs

```diff
@@ -1071,6 +1078,74 @@ internal static byte[] WriteJweHeader(SecurityTokenDescriptor tokenDescriptor)
                 }
             }
         }
+        internal static void WriteActorToken(
+            Utf8JsonWriter writer,
+            SecurityTokenDescriptor tokenDescriptor,
+            bool setDefaultTimesOnTokenCreation,
+            int tokenLifetimeInMinutes)
+        {
+            if (tokenDescriptor == null)
+                throw new ArgumentNullException(nameof(tokenDescriptor));
+
+            var actorTokenDescriptor = CreateActorTokenDescriptor(tokenDescriptor);
+            if (actorTokenDescriptor == null || actorTokenDescriptor.Subject == null)
+                return;
+
+            writer.WritePropertyName(tokenDescriptor.ActorClaimType);
+            WriteJwsPayload(ref writer, actorTokenDescriptor, setDefaultTimesOnTokenCreation, tokenLifetimeInMinutes);
```

Comment:
We can simply this method by just passing the ClaimsIdentity, and the writing the claims into JSON.
Assume that only the claims in the ClaimsIdentity need to be added to the JSON.